docs/use/app_tokens.md

```diff
@@ -80,3 +80,38 @@ Tokens beginning `BRJS.` are system tokens and cannot be overridden within `.pro
 - `BRJS.APP.VERSION`: The app version &mdash; either automatically generated or specified by using the `-v` flag.
 - `BRJS.APP.LOCALE`: If used in an index page will be replaced with the page's locale, otherwise replaced with the app's default locale.
 - `BRJS.BUNDLE.PATH`: A relative path to the directory containing the app's bundles &mdash; this should be used by library code needing to reference bundles since the relative URL will change depending on the versioning mechanism in use by the app.
+
+## Token Format Ambiguity
+
+<div class="alert alert-warning">
+	<p>The token format `@[A-Z\.]+@` is occasionally used by other libraries for things, such as date placeholders, and can lead to ambiguity as to whether or not it's a token that should be replaced by BRJS. In order to prevent apps from being built and deployed with unreplaced tokens BRJS will throw an exception if a token replacement can be found, but this behaviour results in exceptions for libraries where the intention is to use the literal characters rather than a token.</p>
```

Comment:
"if a token replacement can be found" should be "if a token replacement can not be found"